components/homepage-contributor-spotlight/server.css

```
@@ -21,7 +21,7 @@
 }
 
 .homepage-contributor-spotlight__name {
-  width: max-content;
+  max-width: 100%;
```

I would think we should keep `width: max-content`, so it takes as much space as it needs, but without overflowing.

```suggestion
  max-width: 100%;
  width: max-content;
```